public/scripts/extensions/memory/index.js

```diff
@@ -280,14 +284,39 @@ function onMemoryPositionChange(e) {
 function onMemoryPromptWordsForceInput() {
     const value = $(this).val();
     extension_settings.memory.promptForceWords = Number(value);
-    $('#memory_prompt_words_force_value').text(extension_settings.memory.promptForceWords);
+    const display = $('#memory_prompt_words_force_value');
+    display.text(extension_settings.memory.promptForceWords);
+    updateCase(display.next(), extension_settings.memory.promptForceWords);
     saveSettingsDebounced();
 }
 
+function updateCase(span, number) { // other languages can use this function too, adding their own attributes to the elements
```

Comment:
I'd rather make the text display universal than introduce a new subsystem for localizing numeric cases.

`Target summary length (200 words)` => `Target summary length (words): 200`

Reply:
Removed the updateCase() function, changed html structure a bit to use your proposed solution while keeping the English UI exactly the same.
Also, fixed a couple strings in the translation file while at it.